helpers/set_ext.rei

```diff
@@ -0,0 +1,20 @@
+include (module type of {
+  include Set;
+});
+module Make_with_yojson:
+  (
+    K: {
+      type t;
+      let compare: (t, t) => int;
+      let to_yojson: t => Yojson.Safe.t;
+      let of_yojson: Yojson.Safe.t => result(t, string);
+    },
+  ) =>
+   {
+    include (module type of {
+      include Set.Make(K);
+    });
```

Comment:
Do you have any idea what is the difference here? Between 

```reason
include (module type of {
  include Set.Make(K);
});
// and
include (module type of Set.Make(K))
```
I remember is something related to type strengthening but I truly don't understand if it makes any difference here.